src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+// eslint-disable-next-line import/no-cycle
+import { useEffect } from 'react';
+// eslint-disable-next-line import/no-cycle
+import { ErrorType } from '../App';
+
+type Props = {
+  error: ErrorType | null,
+  onError(type: ErrorType | null): void,
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error, onError }) => {
+  useEffect(() => {
+    setTimeout(() => onError(null), 3000);
+  });
```

Comment:
Do not forget add dependecies, or it will be called on each render
```suggestion
  useEffect(() => {
    setTimeout(() => onError(null), 3000);
  }, []);
```